config/cfg_ppx.ml

```diff
@@ -33,12 +33,8 @@ let eval_attr attr =
     let loc = attr.attr_loc in
     (* Printf.printf "\n\nattr name: %S\n\n" attr.attr_name.txt; *)
     match attr.attr_payload with
-    | PStr payload ->
-        let payload = Pprintast.string_of_structure payload in
-        (* NOTE(leostera): payloads begin with `;;` *)
-        let payload = String.sub payload 2 (String.length payload - 2) in
-        (* Printf.printf "\n\npayload: %S\n\n" payload; *)
-        if Cfg_lang.eval ~loc ~env payload then `keep else `drop
+    | PStr [ { pstr_desc = Pstr_eval (e, []); _ } ] ->
+        if Cfg_lang.eval ~loc ~env e then `keep else `drop
```

Comment:
I think this should fix the conflicts with `main`, while keeping these two things for debugging handy.

```suggestion
        (* let e = Pprintast.string_of_expression e in *)
        (* Printf.printf "\n\npayload: %S\n\n" e; *)
        if Cfg_lang.eval ~loc ~env e then `keep else `drop
```